webrtc.html

```diff
@@ -1010,7 +1114,7 @@
             connectivity being established.</p>
 
             <p>When the <dfn id=
-            "dom-peerconnection-updateice"><code>updateIce()</code></dfn>
+            "dom-peerconnection-setconfiguration"><code>setConfiguration()</code></dfn>
             method is invoked, the user MUST run the following steps to process
             the <code><a>RTCConfiguration</a></code> dictionary:</p>
```

Comment:
The steps are in the text below, lines 1121-1207 in the new file.

I think it would be fine to prevent any changes to peerIdentity. We can relax this in the future if needed.